internal/pkg/template/templates/environment/partials/elb-access-logs.yml

```diff
@@ -41,4 +41,60 @@ ELBAccessLogsBucket:
       BlockPublicAcls: true
       BlockPublicPolicy: true
       IgnorePublicAcls: true
-      RestrictPublicBuckets: true
\ No newline at end of file
+      RestrictPublicBuckets: true
+
+ELBAccessLogsBucketCleanerAction:
+  Metadata:
+    'aws:copilot:description': 'A custom resource that empties the ELB access logs bucket'
+  Type: Custom::BucketCleanerFunction
+  Properties:
+    ServiceToken: !GetAtt BucketCleanerFunction.Arn
+    BucketName: !Ref ELBAccessLogsBucket
+
+BucketCleanerFunction:
+  Type: AWS::Lambda::Function
+  Properties:
+    {{- with $cr := index .CustomResources "BucketCleanerFunction" }}
+    Code:
+      S3Bucket: {{$cr.Bucket}}
+      S3Key: {{$cr.Key}}
+    {{- end}}
+    Handler: "index.handler"
+    Timeout: 900
+    MemorySize: 512
+    Role: !GetAtt 'ELBAccessLogsBucketCleanerRole.Arn'
+    Runtime: nodejs16.x
+
+ELBAccessLogsBucketCleanerRole:
+  Metadata:
+    'aws:copilot:description': 'An IAM role {{- if .PermissionsBoundary}} with permissions boundary {{.PermissionsBoundary}} {{- end}} to clean the ELB access logs bucket'
+  Type: AWS::IAM::Role
+  Properties:
+    AssumeRolePolicyDocument:
+      Version: '2012-10-17'
+      Statement:
+        -
+          Effect: Allow
+          Principal:
+            Service:
+              - lambda.amazonaws.com
+          Action:
+            - sts:AssumeRole
+    {{- if .PermissionsBoundary}}
+    PermissionsBoundary: !Sub 'arn:${AWS::Partition}:iam::${AWS::AccountId}:policy/{{.PermissionsBoundary}}'
+    {{- end}}
+    Path: /
+    Policies:
+      - PolicyName: "CleanELBAccessLogs"
+        PolicyDocument:
+          Version: '2012-10-17'
+          Statement:
+            - Effect: Allow
+              Action: "s3:*"
```

Comment:
maybe scope down to just the actions we need 👀 ?

Reply:
i've tried but it seems like the action that we use in the custom resource is not a valid one for IAM...we'll keep it for now until someone has better idea. Should be good enough for now as at least we limit the resource to only this bucket.

Reply:
i think [`HeadBucket`](https://docs.aws.amazon.com/AmazonS3/latest/API/API_HeadBucket.html) is `s3:ListBucket`, [`ListObjectVersions`](https://docs.aws.amazon.com/AmazonS3/latest/API/API_ListObjectVersions.html) is `s3:ListBucketVersions` (though you could add `s3:ListBucket` to be safe), and [`DeleteObjects`](https://docs.aws.amazon.com/AmazonS3/latest/API/API_DeleteObjects.html) is.... `s3:DeleteObject` and `s3:DeleteObjectVersion`?